Gulpfile.js

```diff
@@ -144,6 +144,9 @@ gulp.task('lint-client', function () {
         .src([
             'src/client/runner/automation/playback/scroll.js',
             'src/client/runner/automation/playback/move.js',
+            'src/client/runner/automation/playback/press.js',
```

Comment:
I see you've moved and renamed the `press.js` file. Should update the `lint-client` task